src/components/AmplitudeAnalytics/index.ts

```diff
@@ -7,32 +7,30 @@ import { isProductionEnv } from 'utils/env'
  * Uniswap has two Amplitude projects: test and production. You must be a
  * member of the organization on Amplitude to view details.
  */
-export function initializeAnalytics() {
-  const API_KEY = isProductionEnv() ? process.env.REACT_APP_AMPLITUDE_KEY : process.env.REACT_APP_AMPLITUDE_TEST_KEY
 
-  if (typeof API_KEY === 'undefined') {
-    const keyName = isProductionEnv() ? 'REACT_APP_AMPLITUDE_KEY' : 'REACT_APP_AMPLITUDE_TEST_KEY'
-    console.error(`${keyName} is undefined, Amplitude analytics will not run.`)
-    return
-  }
+const AMPLITUDE_KEY_NAME = isProductionEnv() ? 'REACT_APP_AMPLITUDE_KEY' : 'REACT_APP_AMPLITUDE_TEST_KEY'
+const API_KEY = process.env[AMPLITUDE_KEY_NAME]
 
-  init(
-    API_KEY,
-    /* userId= */ undefined, // User ID should be undefined to let Amplitude default to Device ID
-    /* options= */
-    {
-      // Disable tracking of private user information by Amplitude
-      trackingOptions: {
-        ipAddress: false,
-        carrier: false,
-        city: false,
-        region: false,
-        dma: false, // designated market area
-      },
-    }
-  )
+if (typeof API_KEY === 'undefined') {
+  throw new Error(`${AMPLITUDE_KEY_NAME} is undefined, Amplitude analytics will not run.`)
```

Comment:
you sure we should throw here?

Reply:
if we don't have amplitude analytics set up then amplitude experiment set up will fail. we want amplitude experiment to be set up even on local now, so i think we do want this. i could avoid by skipping local usage of experiment, but then we lose default flags.

Reply:
i feel like we need to start giving people a default set of development keys?

Reply:
i think a console error will let people know it's not working, but in general i think failing gracefully is always best. if there's no API key for analytics, imo things should still work instead of break

Reply:
gotcha, i will figure out a better way to handle this